homeassistant/components/zwave_js/sensor.py

```diff
@@ -328,152 +327,154 @@
 }
 
 
-def convert_dict_of_dicts(
-    statistics: ControllerStatisticsDataType | NodeStatisticsDataType, key: str
+def convert_nested_attr(
+    statistics: ControllerStatistics | NodeStatistics, key: str
 ) -> Any:
-    """Convert a dictionary of dictionaries to a value."""
-    keys = key.split(".")
-    return statistics.get(keys[0], {}).get(keys[1], {}).get(keys[2])  # type: ignore[attr-defined]
+    """Convert a string that represents a nested attr to a value."""
+    data = statistics
+    for _key in key.split("."):
+        if data is None:
+            return None  # type: ignore[unreachable]
+        data = getattr(data, _key)
+    return data
 
 
 @dataclass(frozen=True, kw_only=True)
 class ZWaveJSStatisticsSensorEntityDescription(SensorEntityDescription):
     """Class to represent a Z-Wave JS statistics sensor entity description."""
 
-    convert: Callable[
-        [ControllerStatisticsDataType | NodeStatisticsDataType, str], Any
-    ] = lambda statistics, key: statistics.get(key)
+    convert: Callable[[ControllerStatistics | NodeStatistics, str], Any] = getattr
     entity_registry_enabled_default: bool = False
 
 
 # Controller statistics descriptions
 ENTITY_DESCRIPTION_CONTROLLER_STATISTICS_LIST = [
     ZWaveJSStatisticsSensorEntityDescription(
-        key="messagesTX",
+        key="messages_tx",
```

Comment:
Changing theses keys, will change the unique ID of the entities as it seems, meaning this needs a migration?

Reply:
good point, thanks for catching that